### PR TITLE
create  educational details

### LINKED ISSUE #5: Educational Details

### PR DESCRIPTION
**Pull request by ajstyle2410**
add  master degree
       bachelor degree
        higher education
        secondary education
details

closes #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,66 @@
 </head>
 
 <body>
+    <div class="educatinal-details">
+        <div class="card">
+            <h5 class="card-header">Educational Details</h5>
+            <div class="card-body">
+                <div class="detail">
+                    <div class="row">
```
Comment (shubham-8551) [inline index.html:21]: add col section as Degree and year of passing

File: index.html [hunk 2]
```diff
@@ -13,6 +13,66 @@
...
+                        <div class="col-6">
+                            <h6>Master of Technology in Computer Science </h6>
+                        </div>
+                        <div class="col-6">
+                           
+                            <label class="fw-bolder">  May 2024</label>
```
Comment (shubham-8551) [inline index.html:27]: mark year first and month later

File: index.html [hunk 3]
```diff
@@ -13,6 +13,66 @@
...
+
+                        </div>
+                        <div class="col-12 mb-2">
+                           <strong class="text-capitalize"> pandit shivdatta shastri education foundation's asma institute of management Shivane Pune</strong>
+                        </div>
+                               <hr>
+                        <div class="col-6">
+                            <h6>Bachelore of Technology in Computer Science </h6>
```
Comment (shubham-8551) [inline index.html:35]: If you want you can highlight course without highlighting college name.

Comment (ajstyle2410) [reply index.html]: resolved education page  structure

File: index.html [hunk 4]
```diff
@@ -17,58 +17,42 @@
...
-                   
-
-                </div>
+                <table class="table table-dark table-striped table-hover">
+                    <thead>
+                      <tr>
+                        <th scope="col">Passout Year</th>
+                        <th scope="col">Examination</th>
+                        <th scope="col">Institute</th>
+                        <th scope="col">Sgpa/ Percentage</th>
```
Comment (shubham-8551) [inline index.html:26]: Keep only percentage . If all marks in percentage format

Comment (ajstyle2410) [reply index.html:26]: sgpa are resolved

**Pull request merged**